metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -401,15 +408,16 @@ class TableauConfig(
 
     ingest_multiple_sites: bool = Field(
         False,
-        description="When enabled, ingests multiple sites the user has access to. If the user doesn't have access to the default site, specify an initial site to query in the site property. By default all sites the user has access to will be ingested. You can filter sites with the site_name_pattern property.",
+        description="When enabled, ingests multiple sites the user has access to. If the user doesn't have access to the default site, specify an initial site to query in the site property. By default all sites the user has access to will be ingested. You can filter sites with the site_name_pattern property. This flag is currently only supported for Tableau Server. Tableau Cloud is not supported.",
     )
 
     site_name_pattern: AllowDenyPattern = Field(
         default=AllowDenyPattern.allow_all(),
         description="Filter for specific Tableau sites. "
-                    "By default, all sites will be included in the ingestion. "
-                    "You can both allow and deny sites based on their name using their name, or a Regex pattern. "
-                    "Deny patterns always take precedence over allow patterns. ",
+        "By default, all sites will be included in the ingestion. "
+        "You can both allow and deny sites based on their name using their name, or a Regex pattern. "
+        "Deny patterns always take precedence over allow patterns. "
+        "This property is currently only supported for Tableau Server. Tableau Cloud is not supported. ",
```

Comment:
Not a blocker here, but I do wonder if there's any way we can programmatically determine if we're connecting to tableau server or tableau cloud, to that we can output better error messages when someone tries to use this config with Tableau cloud.

Reply:
Good idea, I checked and found the server_info endpoint: https://tableau.github.io/server-client-python/docs/api-ref#serverinfoitem-class but unfortunately, it doesn't reveal if it's Tableau Server or Tableau Cloud. If you know a different approach, please let me know. Otherwise I'd say we keep it as-is for now.